drake_ros_systems/README.md

```diff
@@ -0,0 +1,155 @@
+1. Activate the drake virtual environment
+1. Build it using Colcon, or using CMake directly
+    
+    **Colcon**
+    1. Make a workspace `mkdir -p ./ws/src`
+    1. `cd ./ws/src`
+    1. Get this code `git clone https://github.com/RobotLocomotion/drake-ros.git`
+    1. `cd ..`
+    1. Build this package `colcon build --packages-select drake_ros_systems`
+    
+    **CMake**
+    1. Manually set `CMAKE_PREFIX_PATH`: `export CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:$AMENT_PREFIX_PATH`
+    1. Get this code `git clone https://github.com/RobotLocomotion/drake-ros.git`
+    1. `cd drake-ros/drake_ros_systems`
+    1. Make a build and install folder to avoid installing to the whole system `mkdir build install`
+    1. `cd build`
+    1. Configure the project `cmake -DCMAKE_INSTALL_PREFIX=$(pwd)/../install ..`
+    1. Build the project `make && make install`
+
+# Running
+
+If you built with `colcon`, then source your workspace.
+
+```
+. ./ws/install/setup.bash
+# Also make sure to activate drake virtual environment
+```
+
+If you built with plain CMake, then source the ROS workspace and set these variables.
+
+```
+. /opt/ros/rolling/setup.bash
+# Also make sure to activate drake virtual environment
+# CD to folder containing `build` and `install` created earlier, commands below use PWD to find correct path
+export LD_LIBRARY_PATH=$(pwd)/install/lib:$LD_LIBRARY_PATH
+export PYTHONPATH=$(pwd)/install/lib/$(python -c 'import sys; print(f"python{sys.version_info[0]}.{sys.version_info[1]}")')/site-packages:$PYTHONPATH
+```
+
+Now you can run C++ examples from the build folder and Python examples from the source folder.
+
+# Examples
+
+Some examples of using these systems are given in the [`example`](./example) folder in two languages: Python and C++.
+
+## RS flip flop
+
+### Overview
+
+Both `rs_flip_flop` and `rs_flip_flop.py` implement an RS flip flop using NOR gates.
+They subscribe to the following topics:
+
+* `/R`
+* `/S`
+
+And publish to the following topics
+
+* `/Q`
+* `/Q_not`
+
+### How To
+
+If built with **colcon** using an isolated build (default), run the C++ example as follows:
+
+```
+./build/drake_ros_systems/example/rs_flip_flop
+```
+
+If built with **cmake** or a non-isolated build of **colcon**, run the C++ example as follows:
+
+```
+./build/example/rs_flip_flop
+```
+
+
+The Python example can always be run from the source folder as follows:
+
+```
+python3 ./example/rs_flip_flop.py
+```
+
+Run these commands in different terminals with your ROS installation sourced to echo the output topics:
+
+```
+ros2 topic echo /Q
+```
+
+```
+ros2 topic echo /Q_not
+```
+
+Run these commands in different terminals with your ROS installation sourced to play with the input topics.
+
+```
+ros2 topic pub /S std_msgs/msg/Bool "data: false"
+ros2 topic pub /S std_msgs/msg/Bool "data: true"
+```
+
+```
+ros2 topic pub /R std_msgs/msg/Bool "data: false"
+ros2 topic pub /R std_msgs/msg/Bool "data: true"
+```
+
+## IIWA Manipulator
```

Comment:
Not necessary for this PR, but I envision these examples and tutorials to be in separate markdown files, linked to from this main README.

Reply:
Done, see `drake_ros_examples` package.